index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Zuwairanajma Web</title>
+    <script src = "https://kit.fontawesome.com/1935d064dd.js" 
+    crossorigin = "anonymous">
+   </script>
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+    <!-- Toolbar Header Section starts-->
+    <header> 
+        <div id= "head-wrap" class="headbar">
+    
+  <nav class = "headbar-menu">
+    <p class=""logo>My Logo</p>
+    <i class="fas fa-bars fa-lg"></i>
+  </nav>
+</div>
+</header>
+<!-- Toolbar Header Section ends -->
+
+<main>
+    <!-- Headline Section Start -->
+    <section class="headline">
+    <div class="container_text">
+        <h1 class="ptext">Hey There. <br/>I'm Zuwairanajma.</h1>
+        <h3 class="stext">I'm a Software Developer</h3>
+        <p class="description">I can help you build a product, feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+            don’t hesitate to contact me.</p>
+            <nav class="icon-wrap">
+                <ul class="navicon">
+               <li><a href="https://github.com/Zuwairanajma"><img src ="PImages/GitHubv1.png" alt = "Github Icon"></a></li>
+                <a href="https://www.linkedin.com/in/zuwaira-sadiq-566b891b0?"><img src ="PImages/Linkedinv2.png" alt = "Linkedin Icon"></a></li> 
+                   <a href="https://twitter.com/@SadiqJuwairiyya"><img src ="PImages/Vector3.png" alt = "Angellist Icon"></a> </li>
+                   <a href="https://twitter.com/@SadiqJuwairiyya"><img src ="PImages/Twitter4.png" alt = "Twitter Icon"></a></li>
+                   <a href="https://twitter.com/@SadiqJuwairiyya"><img src ="PImages/Vector5.png" alt = "Medium Icon"></a></li>
```

Comment:
- [ ] Great job! On using the `ul` for your social icons, but all icons, except for the first one, seem to miss an opening `li` tag. I recommend adding them to not face more errors.